projects/objects/rocks/protos/Rock10cm.proto

```diff
@@ -8,6 +8,7 @@ PROTO Rock10cm [
   field SFRotation rotation         0 1 0 0
   field SFString   name             "rock 10 cm"
   field MFString   texture          "textures/rock.jpg"  # Defines the texture used for the rock.
+  field SFColor    color            1 1 1                # Defines the base color of the rock
```

Comment:
Just a minor note: here the final `.` is missing in the field description

Reply:
I will add a test for this in our test suite :wink: